code/system/application/perception/src/detectlane/detectlane.cpp

```diff
@@ -379,99 +439,183 @@ void DetectLane::setUp()
   m_K = Eigen::MatrixXd(m_nRegions,1);
   m_M = Eigen::MatrixXd(m_nRegions,1);
 
-  m_leftTransformationMatrix = readreadThreeByThreeMatrix(
-          "/home/batko/Desktop/leftTransformationMatrix.txt");
-  m_rightTransformationMatrix = readreadThreeByThreeMatrix(
-          "/home/batko/Desktop/rightTransformationMatrix.txt");
 
+  m_leftTransformationMatrix = ReadMatrix(
+          "/opt/opendlv/share/opendlv/tools/vision/projection/leftCameraTransformationMatrixWarped.csv",3,3);
+  std::cout<<m_leftTransformationMatrix;
+  std::cout<<"\n------\n----\n---\n";
+  m_rightTransformationMatrix = ReadMatrix(
+          "/opt/opendlv/share/opendlv/tools/vision/projection/rightCameraTransformationMatrixWarped.csv",3,3);
+
+  m_scale << m_width / (double)m_outputWidth, m_height / (double)m_outputHeight, 1;
+ 
+  if(false)
+  {
+    std::cout<<"Input size\n";
```

Comment:
Always put spaces around separators (except before comma, or next to parenthesis)